.github/workflows/nodejs.yml

```diff
@@ -18,6 +18,12 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
```

Comment:
where is this environment variable needed/used?

Reply:
The env var is used by the CodeClimate test reporter.